build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -73,9 +73,12 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                 }
                 else
                 {
-                    // This is an old drop of latest so remove all old files to ensure a clean state
+                    Regex versionFileRegex = new Regex(@"(?<version>\d\.\d\.\d)-(?<release>.*)?");
```

Comment:
I'm not sure I understand.
`<version>` creates a named group, doesn't affect the matching pattern in any way.